docs/source/usage.rst

```diff
@@ -0,0 +1,185 @@
+Usage
+=====
+
+.. contents::
+
+Most common options
+-------------------
+
+The following exemple uses all most common options. 
+It will add a link to your project website in all pages header and show a link to source code aside every documented elements. 
+
+::
+
+    pydoctor \
+        --add-package=(packagedirectory) \
+        --project-name=(name) \
+        --project-url=https://github.com/(user)/(repo)/ \
+        --html-viewsource-base https://github.com/(user)/(repo)/tree/(tag_or_commit_sha) \
+        --make-html \
+        --html-output=apidocs \
+        --project-base-dir "$(pwd)" \
+        --docformat=restructuredtext 
+
+.. note:: This exemple assume you are running the ``pydoctor`` build from Unix and the current directory is the root folder of your package. 
```

Comment:
Does pydoctor work on Windows ? 
We are only testing it on Linux and maybe macOS.

Reply:
No idea